CHANGELOG.md

```diff
@@ -3,11 +3,14 @@
 ## v1.1.1
```

Comment:
So I just realized that we don't have a separate CHANGELOG.md for the icons-svg package. The 1.1.0 version information here (missing metadata) is for the @pxblue/icons-svg package only (version 1.1.1, which is already published).

So we can either make a separate CHANGELOG for the icons-svg package, or I can publish a 1.1.1 version of icons so we can keep them in sync with a single Readme and this change will be for 1.1.2. I'm leaning towards the latter, but what do you think?

Reply:
I personally don't mind either direction, since our icons-mui is at 2.2.1.

I can see that if we want to fix #73 and #82, the first route might make it easier.

Reply:
For now, for the sake of simplicity, I'll publish the icons font package to keep them in sync. And then we can do the bigger refactor next year.

Reply:
Okay, I published 1.1.1 of the icon font, so if you just update package.json and changelog to say 1.1.2 for your changes we should be good.